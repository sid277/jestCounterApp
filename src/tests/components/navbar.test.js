import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../../components/navbar';

describe("Test the navbar file",()=>{
  
  test('should test NavBar component', () => {
    const wrapper = shallow(<NavBar/>);
    expect(wrapper).toMatchSnapshot();
    });

  test('should test NavBar component', () => {
    const wrapper = shallow(<NavBar totalCounters={4}/>);
    const text = wrapper.text();
    expect(text).toBe("4Items");
    });

});