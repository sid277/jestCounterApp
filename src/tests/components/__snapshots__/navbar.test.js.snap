// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the navbar file should test NavBar component 1`] = `
<nav
  className="navbar navbar-light bg-light"
>
  <div
    className="navbar-brand"
  >
    <i
      aria-hidden="true"
      className="fa fa-shopping-cart fa-lg m-2"
    />
    <span
      className="badge badge-pill badge-info m-2"
      style={
        Object {
          "width": 50,
        }
      }
    />
    Items
  </div>
</nav>
`;
